spec/services/ccms/case_add_requestor_xml_blocks_spec.rb

```diff
@@ -24,11 +23,13 @@ module CCMS # rubocop:disable Metrics/ModuleLength
                fee_earner_contact_id: 4_925_152
       end
 
+      let(:proceeding_type) { create :proceeding_type, :with_real_data }
       let(:application_proceeding_type) { legal_aid_application.application_proceeding_types.first }
       let(:respondent) { legal_aid_application.respondent }
       let(:submission) { create :submission, :case_ref_obtained, legal_aid_application: legal_aid_application }
       let(:requestor) { described_class.new(submission, {}) }
       let(:xml) { requestor.formatted_xml }
+      before { legal_aid_application.proceeding_types << proceeding_type }
```

Comment:
I'd try to avoid using `before` to set up objects. I'd suggest:
```ruby
      let(:proceeding_type) { create :proceeding_type, :with_real_data }
      let(:legal_aid_application) do
        create :legal_aid_application,
               :with_everything,
               :with_applicant_and_address,
               populate_vehicle: true,
               with_bank_accounts: 2,
               proceeding_types: [proceeding_type]
      end
```

Reply:
👍